crates/l2/storage/src/store_db/sql.rs

```diff
@@ -0,0 +1,382 @@
+use std::fmt::Debug;
+
+use crate::api::StoreEngineRollup;
+use crate::error::RollupStoreError;
+use ethrex_common::{
+    H256,
+    types::{Blob, BlockNumber},
+};
+
+use libsql::{
+    Builder, Connection, Row, Rows, Value,
+    params::{IntoParams, Params},
+};
+
+pub struct SQLStore {
+    conn: Connection,
+}
+
+impl Debug for SQLStore {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_str("data")
+    }
+}
+
+const DB_SCHEMA: [&str; 9] = [
```

Comment:
It's easy to mess this up since it's simply a string, let's add a test that checks that the resulting schema is the one defined here and only what's defined here.

Reply:
If this ends up being only for debugging like the PR text description implies, disregard this.